webapp-django/crashstats/signature/jinja2/signature/signature_report.html

```diff
@@ -65,7 +65,9 @@ <h2>Signature report for <em>{{ signature }}</em></h2>
         <li><a href="#reports" class="reports" data-tab-name="reports">Reports</a></li>
         <li><a href="#graphs" class="graphs" data-tab-name="graphs">Graphs</a></li>
         <li><a href="#bugzilla" class="bugzilla" data-tab-name="bugzilla">Bugzilla</a></li>
-        <li><a href="#comments" class="comments" data-tab-name="comments">Comments</a></li>
+        {% if request.user.has_perm('crashstats.view_pii') %}
+          <li><a href="#comments" class="comments" data-tab-name="comments">Comments</a></li>
+        {% endif %}
```

Comment:
This is a ui-nicety. Why show a tab the user can't see the data of.

I'd rather we did this in other ways--like maybe the signature report view should determine which tabs can be seen and then we iterate over that? I don't know.

Reply:
I can imagine the version that iterates, but I'm not sure if that would be some Jinja code or a structure setup in the view. It would also make it harder to read. When the templates get too far away from the "View Source" version, I get uncomfortable.

The reason to show the tab would be to allow access to the page, but show the user a message on the Comments page saying "Comments may include personal information, and you aren't allowed to see them, see this page for details".